Xamarin.Forms.Core/Items/ItemsLayoutTypeConverter.cs

```diff
@@ -1,23 +1,51 @@
 ﻿using System;
+using System.Globalization;
 
 namespace Xamarin.Forms
 {
 	[Xaml.TypeConversion(typeof(IItemsLayout))]
 	public class ItemsLayoutTypeConverter : TypeConverter
 	{
+		private const string HorizontalGridPrefix = "HorizontalGrid,";
+		private const string VerticalGridPrefix = "VerticalGrid,";
```

Comment:
don't include the comma in the prefix. you can even drop this const, as you use the value by string later on

Reply:
I re-implemented the parsing. At first identifier is identified and then span is parsed, if string is longer.